packages/react-button/src/components/CompoundButton/useCompoundButtonStyles.ts

```diff
@@ -1,356 +1,224 @@
 import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
-import { Theme } from '@fluentui/react-theme';
 import { buttonSpacing, useButtonStyles } from '../Button/useButtonStyles';
-import { CompoundButtonState, CompoundButtonStyleSelectors, CompoundButtonVariantTokens } from './CompoundButton.types';
+import { CompoundButtonState } from './CompoundButton.types';
 
 const CompoundButtonClassNames = {
   secondaryContent: 'CompoundButton-secondaryContent',
 };
 
-export const makeCompoundButtonTokens = (theme: Theme): CompoundButtonVariantTokens => ({
-  base: {
-    // root tokens
-    height: 'auto',
-    paddingX: buttonSpacing.large,
-    paddingY: buttonSpacing.large,
+const useRootStyles = makeStyles({
+  base: theme => ({
+    gap: buttonSpacing.large,
 
-    // icon tokens
-    iconFontSize: '40px',
-    iconSpacing: buttonSpacing.large,
-    iconHeight: '40px',
-    iconWidth: '40px',
+    height: 'auto',
 
-    // secondary content tokens
-    secondaryContentColor: theme.alias.color.neutral.neutralForeground2,
-    secondaryContentFontSize: theme.global.type.fontSizes.base[200],
-    secondaryContentFontWeight: theme.global.type.fontWeights.regular,
-    secondaryContentGap: buttonSpacing.smaller,
+    [`& .${CompoundButtonClassNames.secondaryContent}`]: {
+      color: theme.alias.color.neutral.neutralForeground2,
+    },
 
-    hovered: {
-      secondaryContentColor: theme.alias.color.neutral.neutralForeground2Hover,
+    ':hover': {
+      [`& .${CompoundButtonClassNames.secondaryContent}`]: {
+        color: theme.alias.color.neutral.neutralForeground2Hover,
+      },
     },
 
-    pressed: {
-      secondaryContentColor: theme.alias.color.neutral.neutralForeground2Pressed,
+    ':active': {
+      [`& .${CompoundButtonClassNames.secondaryContent}`]: {
+        color: theme.alias.color.neutral.neutralForeground2Pressed,
+      },
     },
-  },
+  }),
   small: {
-    paddingX: buttonSpacing.medium,
-    paddingY: buttonSpacing.medium,
-
-    fontSize: theme.global.type.fontSizes.base[300],
-    lineHeight: theme.global.type.lineHeights.base[300],
-  },
-  large: {
-    paddingX: buttonSpacing.larger,
-    paddingY: buttonSpacing.larger,
-
-    secondaryContentFontSize: theme.global.type.fontSizes.base[300],
-  },
-  iconOnly: {
-    maxWidth: '52px',
-    minWidth: '52px',
-  },
-  iconOnlySmall: {
-    maxWidth: '48px',
-    minWidth: '48px',
+    padding: `${buttonSpacing.medium} ${buttonSpacing.medium}`,
```

Comment:
You are right, I think we had these separate because we were making a distinction between `paddingX` and `paddingY` in the legacy approach. I'll change this accordingly.